abe/resource_models/event_resources.py

```diff
@@ -24,6 +24,19 @@
 from abe.helper_functions.sub_event_helpers import create_sub_event, update_sub_event, sub_event_to_full, access_sub_event, find_recurrence_end
 from abe.helper_functions.query_helpers import get_to_event_search, event_query
 
+api = Namespace('events', description='Events related operations')
+
+event_model = api.model('Events_Model', {
```

Comment:
Add a comment here and to Events, that when one is updated the other should be too.

Reply:
When you say "to Events", are you suggesting the document model? Or the class just below this?

Reply:
If I understand correctly, this statement needs to be manually (as we discussed) kept in sync with what I now see is `event_documents.Event`, so I was proposing a comment here and at the definition of `event_documents.Event`.